### PR TITLE
Ignore multi-line annotations, strings and comments

### PR DESCRIPTION
**Pull request by jrha**

### CODE REVIEW

File: pan_indent_checker.py [hunk 1]
```diff
@@ -44,9 +48,20 @@ def supports_color():
...
-def _print_debug(k, v):
-    print(DEBUG_DIVIDER)
-    print(DEBUG_LINE % (k, v))
+def _print_debug(k, v, c=""):
+    print(DEBUG_LINE % (k, c[:3], v))
+
+
+def find_annotation_blocks(text):
+    """Find multi-line annotation blocks in a block of text"""
+    result = []
```
Comment (tofu-rocketry) [inline pan_indent_checker.py:57]: Personal preference would be to use the plural for a list.

File: pan_indent_checker.py [hunk 2]
```diff
@@ -70,46 +85,62 @@ def main():
     colorama_init(strip=(not supports_color()))
 
     with open(args.input, 'r+') as file_input:
-        for line_number, line_raw in enumerate(file_input):
+        # Find annotations and reset pointer
+        annotations = find_annotation_blocks(file_input.read())
+        file_input.seek(0)
+
+        # Iterate over lines
+        for line_number, line_raw in enumerate(file_input, 1):
```
Comment (tofu-rocketry) [inline pan_indent_checker.py:93]: How has the start value of the enumeration changed without seemingly needing a change to the looping? Was it a bug originally?

**Review by tofu-rocketry — COMMENTED** [review]
:chocolate_bar:

File: pan_indent_checker.py (hunk 2 above)
Comment (jrha) [reply pan_indent_checker.py:93]: Oh, yes it was, but wasn't spotted before as the numbering was only used for output.

File: pan_indent_checker.py (hunk 1 above)
Comment (jrha) [reply pan_indent_checker.py:57]: Meh.

Comment (tofu-rocketry) [reply pan_indent_checker.py:57]: Humph.

Comment (jrha) [reply pan_indent_checker.py:57]: Put another way, this function was copied from Panlint and I'd like it to stay the same.

**Review by tofu-rocketry — APPROVED** [review]
:shipit:

**Comment by jrha** [comment]
:goberserk:

**Pull request merged**